wallet/src/ui/app/pages/home/receipt/index.tsx

```diff
@@ -43,35 +39,50 @@ function ReceiptPage() {
     const ctaLinks = tranferType === 'nft' ? '/nfts' : '/';
     const linkTo = tranferType ? ctaLinks : '/transactions';
 
+    const navigate = useNavigate();
+    const closeReceipt = useCallback(() => {
+        navigate(linkTo);
+    }, [linkTo, navigate]);
+
     if (!txDigest && txResults && !txnItem) {
         return <Navigate to={linkTo} replace={true} />;
     }
 
+    //TODO : add more transfer types and messages
+    const transfersTxt = {
+        Call: {
+            sender: 'Minted Successfully',
```

Comment:
Why is this minted? Not all calls are mints?

Reply:
Just copied the design

Reply:
I'm guessing that's not the intent of the design. cc @mystie711